scripts/bagging_variables.sh

```diff
@@ -12,12 +12,14 @@ export BAG_ALWAYS="/odom /absodom /clock /tf /tf_static"
 # Define the directory that the bags will be stored in
 export BAG_DIR=~/bags
 
+# Which camera image type to use
+export CAM_SUFFIX="image_rect_color"
```

Comment:
Please note that a command line argument should be added to `mil_common/scripts/bag.sh` for the `CAM_SUFFIX` variable. This can be copied from the sections that address the `BAG_ALWAYS` or `BAG_HOME` variables. Also, I would prefer that it be called `BAG_CAM_SUFFIX` to keep with my naming convention in this file as well as to have the comment begin with the word "Define" (sorry to nit pick).